curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/6153a3485f0b20591d26d2a1.md

```diff
@@ -7,14 +7,14 @@ dashedName: step-19
 
 # --description--
 
-Smooth out your images a bit by giving the `#gallery img` selector a `border-radius` property of `10px`.
+Smooth out your images a bit by giving the `.gallery img` selector a `border-radius` property with `10px` set as the value.
```

Comment:
What do you think of?
```suggestion
To smooth out your images. Give `border-radius` property with a value of `10px` to `.gallery img` selector.
```

Reply:
The first part of the sentence sounds incomplete to me. I also don't really see an issue with the sentence as it is.

Reply:
Ya this one is weird 😅